openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>1)</p><p> </p><p>I'd remove the output from this cell folks might get confused as to why it's CPU only :P</p><p> </p><p>2)</p><p> </p><p>[enhancement]</p><p> </p><p>You could possibly add a "performance consideration" section - something along the lines of "if you're running gas phase, then we suggest setting <code>OPENMM_CPU_THREADS</code> to 1". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Added this!</p>